code/controllers/subsystem/retirement.dm

```diff
@@ -0,0 +1,71 @@
+// retirement.dm
+// Copyright 2021 Patrick Meade.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+//------------------------------------------------------------------------------
+
+SUBSYSTEM_DEF(retirement)
+	flags = SS_NO_FIRE            // no need to fire this after initialization
+	init_order = INIT_ORDER_LAST  // as long as it's after INIT_ORDER_DBCORE...
```

Comment:
`SSdbcore.Initialize()` only checks if the schema is valid. The actual connection happens at the start of `world/New()`